### ISSUE TITLE
somehow wrong shader got set on PBS_DualSidedMetallic in specific world

### ISSUE THREAD

**Issue by zahndy** [Yellow-Dog-Man/Resonite-Issues#674]
### Describe the bug?

PBS_DualSidedMetallic does not behave as expected, 
The only blendmode that shows color is AlphaBlend, but that one also adds shading to parts of the texture that are full alpha.
AlphaClip and Opaque show as translucent white with AO visible.

### To Reproduce

load into "Krugers Office" `resrec:///U-zahndy/R-d51a414f-3a44-43bd-bdec-ee0c154779e2`
Equip devtip, 
spawn out a PBS_DualSidedMetallic.
set to Opaque.
Change the AlbedoColor.
observe how it does not change the color of the material.

### Expected behavior

for the blendmodes to show the proper alpha mode.
and that the AlbedoColor affects the material.

### Screenshots

![20231114083534_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/f0999977-a714-4d01-8660-c641f5ce2c85)
![20231114083550_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/c1b5edda-12df-42a3-a581-90d24fdeb282)
![20231114083556_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/317663d7-f37a-4716-bf46-9f495a057b20)
![20231114083806_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/6c3f9f8f-5c97-4aea-9217-9b6610e4480b)

### Resonite Version Number

2023.11.13.1343

### What Platforms does this occur on?

Windows

### What headset if any do you use?

vive, index, desktop

### Log Files

[HORIZON7 - 2023.11.13.1343 - 2023-11-14 08_53_51.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13346042/HORIZON7.-.2023.11.13.1343.-.2023-11-14.08_53_51.log)

**Comment by lxw404** [comment]
I've tried the steps listed above on multiple meshes and the material does indeed change color.  I was unable to reproduce this even with changing blend modes.

**Comment by zahndy** [comment]
Thanks for pointing that out, its specific to that world.
Its driving me mad though.
I can't reproduce it in gridspace, even when I copy the sun and skybox over.
Disabling everything else in the world doesn't change the shading, it seems to be completely unaffected by skybox or sun changes.
I can't figure out what is actually causing this

**Comment by lxw404** [comment]
You've tried copying the object that's misbehaving over into a grid world too?

**Comment by zahndy** [comment]
The object the material is being applied to doesn't really matter, the MaterialOrb is unlit-white the moment I spawn out PBS_DualSidedMetallic in that world.
If I spawn the material in gridspace it looks fine, if it set it to red and save it, spawn it out in that specific world it looks white again.

Even more weird, if I spawn out a PBS_DualSidedSpecular, it looks correct.
![20231114132941_1](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/7ebd32a3-4209-4fbc-ba5b-2e2e6dcae74e)

**Comment by Nytra** [comment]
Yeah so I have loaded up the same world (Krugers Office), spawned out a PBS_DualSidedMetallic, in the material opened the ref to the _regular shader and then deleted it in the inspector. It fixed the problem of the material being broken.

**Comment by Nytra** [comment]
I have seen another new report of the PBS_DualSidedMetallic shader being broken in a world. The fix is still the same (delete the StaticShader component for the material). https://discord.com/channels/1040316820650991766/1183467648470753401

**Comment by Nytra** [comment]
I'm not sure if closing this issue is right because the problem still happens, we just don't know what causes it / haven't been able to get a replication method for it. It might be caused by mods in some way but that is not confirmed. It is possible that by keeping this open people can provide more info in the future on how to trigger it.

**Comment by zahndy** [comment]
I've been trying to get back to this but the issue is that I've worked on this world for over a month, between 2 computers. resulting in >50 logs where the world name changed at least 4 times, and I have no clue what to look for.
The mods I used should not affect content other than the anisotropic setting on textures:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/662702/41127cd4-8388-40bd-8326-076269cda9c3)

but If someone else that joined had different mods, could that also affect the world content?

**Comment by Nytra** [comment]
Possibly another instance of this still happening https://discord.com/channels/1040316820650991766/1235775992765743164

**Comment by shiftyscales** [comment]
A different instance of this issue seems to have happened with a different shader in #2516. Seeking input from @Geenz on both.

**Comment by GunnarO0** [comment]
I recently had the same issue pop up with the OverlayFresnel material so I figured I'd add onto this. The only thing I can remember doing is importing a .wav file and saving my grid world. Not sure how that would be related and I can't reproduce it.
Deleting the shader component brought back that material type to it's original state.
<img src="https://github.com/user-attachments/assets/e3070bd4-09a6-449a-93a6-c2c764adada4" width=60% height=60%><img src="https://github.com/user-attachments/assets/4fc618fd-6eda-4e54-9f2c-7a44a334e877" width=40% height=40%>
